client/src/tests/e2e/contribuer.spec.ts

```diff
@@ -81,6 +82,14 @@ test.describe("Basic form submission", () => {
     const publishedUrl = page.locator("form [name=publishedUrl]");
     await publishedUrl.fill(publishedUrlText);
     expect(await publishedUrl.inputValue()).toBe(publishedUrlText);
+    // "Mots clés" section
```

Comment:
(Nit) Ligne vide manquante

```suggestion

    // "Mots clés" section
```